PusherServer.Tests/AcceptanceTests/Trigger.cs

```diff
@@ -415,4 +415,92 @@ public async Task It_should_expose_a_multiple_event_ids_when_publishing_to_multi
             Assert.AreEqual(channels.Length, result.EventIds.Count);
         }
     }
+
+    [TestFixture]
+    public class When_requesting_channel_attributes
+    {
+        [Test]
+        public async Task It_should_return_channel_attributes()
+        {
+            var channelName = GetPublicChannelName();
+            var pusher = ClientServerFactory.CreateServer();
+            var channels = new List<string>() { channelName };
+            var options = new TriggerOptions
+            {
+                Info = new List<string>() { "subscription_count" },
+            };
+            await ClientServerFactory.CreateClientAsync(pusher, channelName).ConfigureAwait(false);
+
+            var result = await pusher.TriggerAsync(channels.ToArray(), "event", "data", options).ConfigureAwait(false);
+
+            Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
+            Assert.AreEqual(1, result.ChannelAttributes[channelName].subscription_count);
+        }
```

Comment:
Currently, the only difference between with test and `It_should_return_subscription_count_for_any_channel` is that this test uses an array of channels that has only one channel and the other test uses one channel directly. So right now, these tests are kinda duplicates.

My suggestion is to change this one to test getting channel attributes for multiple channels:

```suggestion
        public async Task It_should_return_channel_attributes_for_multiple_channels()
        {
            var channelName = GetPublicChannelName();
            var pusher = ClientServerFactory.CreateServer();
            var channels = new List<string>() { 
                GetPublicChannelName(),
                GetPublicChannelName(); 
            };
            var options = new TriggerOptions
            {
                Info = new List<string>() { "subscription_count" },
            };
            await ClientServerFactory.CreateClientAsync(pusher, channels[0]).ConfigureAwait(false);
            await ClientServerFactory.CreateClientAsync(pusher, channels[1]).ConfigureAwait(false);

            var result = await pusher.TriggerAsync(channels.ToArray(), "event", "data", options).ConfigureAwait(false);

            Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
            Assert.AreEqual(2, result.ChannelAttributes.Count);
            Assert.AreEqual(1, result.ChannelAttributes[channels[0]].subscription_count);
            Assert.AreEqual(1, result.ChannelAttributes[channels[1]].subscription_count);
        }
```

Reply:
Done!